timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -217,6 +217,40 @@ types:
       LeaderTimes:
         fields:
           leaderTimes: map<Namespace, LeaderTime>
+      TimestampName:
+        alias: string
+        safety: safe
```

Comment:
To wrap or not to wrap, I decided because we are putting this in maps it's better to have a name